govcd/vm.go

```diff
@@ -766,3 +766,80 @@ func (vm *VM) AnswerQuestion(questionId string, choiceId int) error {
 	return nil
 
 }
+
```

Comment:
These two methods seems is duplication from vapp metadata function. I recommend to one common function which just uses different url to edit meta data for vapp or vm.

Reply:
Where would I put that consolidated function? govcd/vdc.go?

Reply:
I think govcd/vapp.go

Reply:
Fixed.